src/features/Apiexplorer/Schema/RecursiveContent/StreamTypesObject/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { Suspense } from 'react';
+import StreamTypesHeader from './StreamTypesHeader';
+import StreamTypesBody from './StreamTypesBody';
+import SourceButton from '../../SourceButton/SourceButton';
+import styles from '../../Schema.module.scss';
+
+type TStreamTypesObject = {
+  definitions: {
+    stream_types: {
+      description: string;
+      type: string;
+      enum;
```

Comment:
does enum not have a type here?

Reply:
Its same as `StreamTypesBody`, let me check if we can grab it from API types itself that way it won't become repetitive and statically typed